components/automate-ui/src/app/pages/infra-proxy/org-create-edit-modal/org-create-edit-modal.component.html

```diff
@@ -0,0 +1,32 @@
+<chef-modal [visible]="visible" (closeModal)="closeEvent()">
+  <h2 slot="title">Create New Org</h2>
+  <div class="flex-container">
+    <form [formGroup]="createForm">
+      <div class="name-margin">
+        <chef-form-field>
+          <label>
+            <span class="label">Org Name <span aria-hidden="true">*</span></span>
+            <input chefInput formControlName="name" type="text" id="name-input" data-cy="full-name" autocomplete="off">
+          </label>
+        </chef-form-field>
+        <chef-form-field>
+          <label>
+            <span class="label">Admin User <span aria-hidden="true">*</span></span>
+            <input chefInput formControlName="admin_user" type="text" id="admin-input" data-cy="admin-user" autocomplete="off">
+          </label>
+        </chef-form-field>
+        <chef-form-field>
+          <label>
+            <span class="label">Admin Key <span aria-hidden="true">*</span></span>
+            <textarea name="admin_key" id="admin_key" formControlName="admin_key" cols="57" rows="10" autofocus></textarea>
+          </label>
+        </chef-form-field>
```

Comment:
Allow all users access to Infra Views option
 - On the orgs settings page, add a toggle to allow all authenticated users access to the API or just admins. Would be greyed out for non-admins. Defaults to admins only.